modules/ROOT/pages/connecting-to-salesforce.adoc

```diff
@@ -34,13 +34,14 @@ After your styling changes for the portal are saved, the AEH Portal Admin team i
 
 == Before You Begin Connecting to Salesforce
 
-* Read the instructions in xref:prerequisites.adoc[Prerequisites].
-* Ensure that you are a System Admin in the Salesforce organization that you are connecting with.
+* Read the instructions in xref:before-you-begin-with-api-experience-hub.adoc[].
+* Ensure that you are a System Admin or have the AEH_Anypoint permission set in the Salesforce organization that you are connecting with.
```

Comment:
Is System Admin the same as System Administrator on line 6?

Reply:
yes